libs/sql-schema-describer/src/postgres.rs

```diff
@@ -799,9 +799,13 @@ impl<'a> SqlSchemaDescriber<'a> {
                         ColumnTypeFamily::DateTime => {
                             match default_string.to_lowercase().as_str() {
                                 "now()"
+                                | "now():::timestamp"
+                                | "now():::timestamptz"
+                                | "now():::date"
                                 | "current_timestamp"
                                 | "current_timestamp():::timestamp"
-                                | "current_timestamp():::timestamptz" => DefaultValue::now(),
```

Comment:
Removal of this one is intentional?

Reply:
It's hiding from you in the line below

Reply:
Indeed. Pattern matching did not anticipate it could change...